File: app/src/main/java/com/asuka/androidopensourcelibrarystudydemo/view/RetrofitRxjavaActivity.java
package com.asuka.androidopensourcelibrarystudydemo.view;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.asuka.androidopensourcelibrarystudydemo.databinding.ActivityRetrofitRxjavaBinding;

public class RetrofitRxjavaActivity extends AppCompatActivity {

    private ActivityRetrofitRxjavaBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding= ActivityRetrofitRxjavaBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
//        setContentView(R.layout.activity_retrofit_rxjava);


    }
}

File: app/src/main/java/com/asuka/androidopensourcelibrarystudydemo/view/activity/RetrofitRxjavaActivity.kt
package com.asuka.androidopensourcelibrarystudydemo.view.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.asuka.androidopensourcelibrarystudydemo.databinding.ActivityRetrofitRxjavaBinding

class RetrofitRxjavaActivity : AppCompatActivity() {
    private lateinit var binding: ActivityRetrofitRxjavaBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityRetrofitRxjavaBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}